quasar.conf.js

```diff
@@ -68,6 +68,10 @@ module.exports = function (ctx) {
             }
         },
         devServer: {
+            watchFiles: [
+                '../freedom-components/f-table',
```

Comment:
Same here

Reply:
Will update how we include on package json once the freedom components are tested and merged